src/context/RSVPContext.js

```diff
@@ -0,0 +1,43 @@
+import {createContext, useState} from "react";
+// import {db} from "../firebase";
```

Comment:
Are these comments necessary?

Reply:
Not necessary - can uncomment them. I commented them out when I was trying to pinpoint where the issue was when handleSubmit was not working.